### ISSUE TITLE
command line tool to generate report based on job statistics

### ISSUE THREAD

**Issue by foreveremain** [GreenAlgorithms/green-algorithms-tool#18]
It would be great to be able to plug data into a command line tool that provides the same report as the web tool. 

Is that possible ?

**Comment by sashavor** [comment]
I have a similar question! I am trying to extract the relevant parts of app.py to calculate the statistics I need, but it's not working (and taking a lot of manual work)

**Comment by Llannelongue** [comment]
Hey, just confirming what you are trying to do: you would like to send requests to the website from the command line to do the calculations? Or are you looking at something local, where the data is loaded on your disk and everything happens locally?

**Comment by sashavor** [comment]
Either way works for me! I think locally is simpler right?

**Comment by Llannelongue** [comment]
Indeed, local is simpler. This is a good idea, and I'll try to add the code for that here. 

But if you want it now, you can give a look to [this project](https://github.com/Llannelongue/GreenAlgorithms4HPC) I've been working on. In particular [this file](https://github.com/Llannelongue/GreenAlgorithms4HPC/blob/main/GreenAlgorithms_global.py). The code is doing more than what you need, but if you replace the part about SLURM query by a simple input, it should do the job nicely?

**Comment by foreveremain** [comment]
looks like it should.. I assumed you'd have probably have created a tool
like this first :)

**Comment by Nanoseb** [comment]
Hi @Llannelongue,
That's great, I did not realise such project existed so I implemented my own a couple of months ago: https://github.com/Nanoseb/gqueue
Once I have some time I will try to contribute to [GreenAlgorithms4HPC](https://github.com/Llannelongue/GreenAlgorithms4HPC).
Cheers and thanks for your work on this topic, it has been a real eye opener for me.
